Gemfile.lock

```diff
@@ -8,56 +8,56 @@ GIT
 
 GIT
   remote: https://github.com/guidance-guarantee-programme/princely.git
-  revision: 4ea8b6be15257b5afc3cfdb1767b8d8c75c72750
+  revision: 9b7170d0350e537da4491958f1977ee1a514d34b
   branch: remove-alias-method-chain
   specs:
-    princely (2.0.2)
+    princely (2.0.3)
```

Comment:
Actually, spoke too soon. Those changes haven't been released, so we can ditch our fork but still need to pin.